account_ux/i18n/es.po

```diff
@@ -426,6 +426,13 @@ msgstr ""
 msgid "Source Document"
 msgstr ""
 
+#. module: account_ux
```

Comment:
remove translations, leave copy to send to PO so they can do the translation from transifex